eldritch/monsters.py

```diff
@@ -97,6 +97,11 @@ def has_attribute(self, attribute, state, char):
       return state_override
     return attribute in self._attributes
 
+  def get_movement(self, state):
+    ancient_modifier = state.ancient_one.get_modifier(self, "movement")
```

Comment:
Let's use get_override instead of get_modifier here (get_modifier is for numeric values). See also the thing above about using the state's get_override instead of using get_modifier.

Reply:
Done!